src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -236,7 +235,7 @@ export class JailListComponent implements OnInit {
         if (this.toActivatePool) {
           this.entityList.getData();
         }
-        this.entityList.snackBar.open("Successfully activate pool " + this.selectedPool , 'close', { duration: 5000 });
+        this.entityList.dialogService.Info(T('Jail Activated'), "Successfully activate pool <i>" + this.selectedPool + "</i>", '500px', 'info', true);
```

Comment:
```suggestion
        this.entityList.dialogService.Info(T('Jail Activated'), "Pool <i>" + this.selectedPool + "</i> is active.", '500px', 'info', true);
```
grammo in original text, suggested small rework